app/controllers/api/v1/field_guides_controller.rb

```diff
@@ -8,7 +8,7 @@ class Api::V1::FieldGuidesController < Api::ApiController
 
   schema_type :json_schema
 
-  before_filter :set_language_if_missing, only: [:index]
+  before_action :set_language_if_missing, only: [:index]
```

Comment:
https://github.com/rubocop/rubocop-rails/issues/343 I ran into this as well. I think this sniff can be "ignored" since `index` is a known controller method

Reply:
Feel free to 'resolve' any comments you don't want to fix. Alternatively if we don't want this cop enforced at all then we can globally turn it off.